FFI/FFI.php

```diff
@@ -30,7 +30,7 @@ class FFI
          *
          * @param string $code The collection of C declarations.
          * @param string|null $library DSO library.
-         * @return CData|mixed
+         * @return CData|FFI|mixed
```

Comment:
```suggestion
         * @return FFI
```

https://github.com/JetBrains/phpstorm-stubs/pull/777#issue-403991092
> FFI::cdef returns an instance of FFI in cases where declarations are more than one.

Actually it always returns \FFI even with empty $code.
https://github.com/php/php-src/blob/fee614f66bfb72384c7c6f316cad0c5c68fa9c0c/ext/ffi/ffi.c#L2932-L2940
https://github.com/php/php-src/blob/fee614f66bfb72384c7c6f316cad0c5c68fa9c0c/ext/ffi/ffi.c#L5008-L5009

```
 % php -r "var_dump(\FFI::cdef(''));"
object(FFI)#2 (0) {
}
```

Reply:
Yep, its true.